### PR TITLE
OSDOCS-8241: adds KI to RNs 4.14

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.14

Issue:

Link to docs preview:
https://66517--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-known-issues:~:text=AWS%20M4-,and,-C4%20instances%20may

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1971,6 +1971,22 @@ As a result, if you apply a performance profile, all nodes in the cluster reboot
...
+
+* For clusters that run on {rh-openstack} that you upgrade to 4.14 and have root volume availability zones, you must converge control plane machines onto one server group before you can enable control plane machine sets. To make the required change, follow the instructions in link:https://access.redhat.com/solutions/7013893[OpenShift on OpenStack with Availability Zones: Invalid Compute ServerGroup setup during OpenShift deployment]. (link:https://issues.redhat.com/browse/OCPBUGS-13300[*OCPBUGS-13300*])
+
+* For clusters that run on {rh-openstack} and that you upgrade to version 4.14, when compute zones are configured with at least one zone, root volumes must now also be configured with at least one zone. If this configuration change does not occur, a control plane machine set cannot be generated for your cluster.
```
Comment (opayne1) [inline release_notes/ocp-4-14-release-notes.adoc:1984]:
```suggestion
* For clusters that run on {rh-openstack} upgradable to version 4.14 and compute zones configured with at least one zone, root volumes must now also be configured with at least one zone. If this configuration change does not occur, a control plane machine set cannot be generated for your cluster.
```

**Review by opayne1 — COMMENTED** [review]
A couple nits/suggestions!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1971,6 +1971,22 @@ As a result, if you apply a performance profile, all nodes in the cluster reboot
...
+
+* There is a known issue in this release which prevents installing a cluster on Alibaba Cloud using installer-provisioned infrastructure. Installing a cluster on Alibaba Cloud is a Technology Preview feature in this release. (link:https://issues.redhat.com/browse/OCPBUGS-20552[*OCPBUGS-20552*])
+
+* From {product-title} {product-version} GA onwards, global IP forwarding is disabled on OVN-Kubernetes based cluster deployments. This was done to prevent undesirable effects for cluster administrators with nodes acting as routers. OVN-Kubernetes now enables and restricts forwarding on a per-managed interface basis.
++
+You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `gatewayConfig.ipForwarding` specification in the `Network` resource. Specify `Restricted` to forward all traffic related to OVN-Kubernetes only. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. For upgrades to 4.14, the default is `Global`. (link:https://issues.redhat.com/browse/OCPBUGS-3176[*OCPBUGS-3176*]) (link:https://issues.redhat.com/browse/OCPBUGS-16501[*OCPBUGS-16051*])
```
[the remark this answers is not in the document]
Comment (JoeAldinger) [reply release_notes/ocp-4-14-release-notes.adoc:1980]: @ahardin-rh

**Pull request merged**